examples/.metadata

```diff
@@ -4,7 +4,7 @@
 # This file should be version controlled.
 
 version:
-  revision: 85684f9300908116a78138ea4c6036c35c9a1236
+  revision: 2ad6cd72c040113b47ee9055e722606a490ef0da
```

Comment:
not sure if I should be updating this file or not

Reply:
Doesn't matter much, it would be replaced when someone runs it with a different version of Flutter